examples/template-svelte-ts/src/App.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  import { useMachine } from '@xstate/svelte';
+  import { toggleMachine } from './toggle.machine';
+  const { state, send } = useMachine(toggleMachine);
+  $: active = $state.matches('active');
+  $: ({ count } = $state.context);
+</script>
+
+<main>
+  <h1>XState Svelte Template</h1>
+  <h2>Fork this template!</h2>
+  <button on:click={() => send('TOGGLE')}>
+    Click me ({active ? '✅' : '❌'})
+  </button>
+  <code>
+    Activated <strong>{count}</strong> times
```

Comment:
Instead of `Toggled` I used the word `Activated`, because the machine increases the counter only on activation. Which is nice that the machine is asymmetric.

The word `toggle` means to me that the counter would increase _every time_ I click on the button. My English is poor however, so I am happy to change it back to `Toggled`.